src/Cli.Tests/Snapshots/InitTests.VerifyCorrectConfigGenerationWithNestedMutationOptions_09cf40a5c545de68.verified.txt

```diff
@@ -0,0 +1,34 @@
+﻿{
+  DataSource: {
+    Options: {
+      container: testcontainer,
+      database: testdb,
+      schema: test-schema.gql
+    }
+  },
+  Runtime: {
+    Rest: {
+      Enabled: false,
+      Path: /api,
+      RequestBodyStrict: true
+    },
+    GraphQL: {
+      Enabled: true,
+      Path: /graphql,
+      AllowIntrospection: true
```

Comment:
Do we get the snapshots for each DataRow being tested?

Reply:
Yes